src/middleware/errorHandler.js

```diff
@@ -1,12 +1,21 @@
 // Import dependencies
+const _ = require("lodash")
 const { serializeError } = require("serialize-error")
 
 // Import logger
 const logger = require("@logger/logger")
 
 function errorHandler(err, req, res, next) {
   if (!err) return next()
-  const errMsg = `${new Date()}: ${JSON.stringify(serializeError(err))}`
+  const containsSensitiveInfo = !_.isEmpty(err.config?.headers?.Authorization)
+  const serialisedErr = serializeError(err)
+  // NOTE: If the error is an axios error or a network error,
+  // the token used (if any) will be present on `request` + `config`.
+  // Hence, we omit both of these.
+  const sanitisedErr = containsSensitiveInfo
+    ? _.omit(serialisedErr, ["request", "config"])
+    : serialisedErr
+  const errMsg = `${new Date()}: ${JSON.stringify(sanitisedErr)}`
```

Comment:
Style: iirc, our cloudwatch already has a date time attached when we view it from the logs.. if thats the case, do you think we can omit this line then?

Reply:
good point but this is in our timezone vs cloudwatch is in UTC :sadge: because of this, i'd rather keep so i dun have to translate timezones 
![Screenshot 2023-05-09 at 6 06 13 PM](https://github.com/isomerpages/isomercms-backend/assets/44049504/5707206b-8f5b-4a8f-a626-7de1b430ab18)

Reply:
<img width="499" alt="Screenshot 2023-05-09 at 6 15 49 PM" src="https://github.com/isomerpages/isomercms-backend/assets/42832651/4fbc56f1-de05-400c-b2f9-08a1ae789609">
Hmm no leh I think displays both